build-system/tasks/presubmit-checks.js

```diff
@@ -1439,36 +1435,26 @@ function isMissingTerms(file) {
 /**
  * Check a file for all the required terms and
  * any forbidden terms and log any errors found.
- * @return {!Promise}
  */
-function presubmit() {
+async function presubmit() {
```

Comment:
```suggestion
function presubmit() {
```

Reply:
This is `async` in order to satisfy the `gulp` task runner. I've updated the comment and task description.